src/UserSwitch/UserSwitch.tsx

```diff
@@ -121,9 +123,11 @@ const UserSwitch: FC<Props> = ({
         {renderAvatar(currentMember)}
 
         <div>
-          <Typography variant='h6' noWrap>
-            {memberName}
-          </Typography>
+          {!isMobile && (
+            <Typography variant='h6' noWrap>
+              {memberName}
+            </Typography>
+          )}
```

Comment:
This name display is the one displayed inside the Popover. If you were trying to not show the name of the user next to the avatar, this is not the right place. You might be looking for line 170. See the screenshot below
<img width="739" alt="Screenshot 2024-01-05 at 15 26 55" src="https://github.com/graasp/graasp-ui/assets/39373170/01e581a2-1637-4a10-8dbf-bd313897734e">